cs/test/TestUtils.cs

```diff
@@ -138,19 +138,20 @@ internal static IDevice CreateTestDevice(DeviceType testDeviceType, string filen
 #if NETSTANDARD || NET
                     if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))    // avoids CA1416 // Validate platform compatibility
 #endif
-                        device = new LocalStorageDevice(filename, preallocateFile, deleteOnClose, disableFileBuffering, capacity, recoverDevice, useIoCompletionPort);
+                        device = new LocalStorageDevice(filename, preallocateFile, deleteOnClose, disableFileBuffering, capacity, recoverDevice, useIoCompletionPort, omitSegmentIdFromFilename: omitSegmentIdFromFilename);
```

Comment:
why is omitSegmentIdFromFileName is both constructor and Initialize? which one take precedence if they put true in contructor and omit (default false) in the Initialize.